pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +405,15 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetInsightDataRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
```

Comment:
I think the application id is optional.
If it was specified we find the data for only the specified one.
If it was empty, we find the data for the whole project.